src/ftva_etl/metadata/digital_data.py

```diff
@@ -17,3 +17,17 @@ def get_uuid(dd_record: dict) -> UUID | str:
 
 def get_media_type(dd_record: dict) -> str:
     return dd_record.get("media_type", "")
+
+
+def get_dcp_info(dd_record: dict) -> dict | None:
+    dcp_info = {}
+    file_type = dd_record.get("file_type", "")
+    folder_name = dd_record.get("file_folder_name", "")
+    if file_type == "DCP":
```

Comment:
I think this logic does not quite match the specs, which have separate handling of each element.  The Asset Type specs say
> If folder_name contains “DCP”, set value to “Exhibition Copy”
but the specs for Folder Name and Sub Folder Name do not require that `folder_name` contains "DCP".

I've raised the question in Slack; leaving this as a regular comment, not a change request, until that's clarified.